Added Minecraft 1.16.1 assets

DrakoTrogdor opened the pull request:
Added Minecraft 1.16.1 assets and updated Readme.

rom1504 commented:
did you make any change to minecraft jar extractor ? if yes please open a PR there

DrakoTrogdor commented:
I made a ton of changes, but they were very hackish, I couldn't get it past the extraction, so I deleted all that code, extracted it manually, then edited the rest to remove the "minecraft:" namespace it was trying to save the items and blocks too. I can post my .js, but it should in no means be merged.

DrakoTrogdor commented:
I have created a PR for minecraft-jar-extractor.
